specification/sphere/Sphere.Management/image.tsp

```diff
@@ -71,7 +73,7 @@ model ImageProperties {
 
 @doc("Image upload request body.")
 model ImageUploadRequestBody {
-  @doc(".")
+  @doc("Image as a UTF-8 encoded base 64 string.")
   images: string;
```

Comment:
>images [](http://example.com/codeflow?start=2&length=6)

Should be singular ("image")

Reply:
removed this in latest commit - model was not needed